tests/test_forms.py

```diff
@@ -1534,6 +1534,10 @@ def validate_slot2(
             return {
                 "slot2": "validated_value",
             }
+        if tracker.get_slot("slot1") == "validated_value":
```

Comment:
Why are these new lines necessary?

Reply:
You should create a separate test for this use-case because as far as I can see the scenario when `slot2` would be None is never tested.

Reply:
Okay, I have reverted it back as what I wanted to check there had already been included in the new test that I added.